apps/bmd/src/components/CandidateContest.tsx

```diff
@@ -313,6 +324,12 @@ const CandidateContest = ({ contest, parties, vote, updateVote }: Props) => {
                   const party =
                     candidate.partyId &&
                     findPartyById(parties, candidate.partyId)
+                  let prefixAudioText = ''
+                  if (isChecked) {
+                    prefixAudioText = 'Selected, '
```

Comment:
Space after comma is unnecessary as it is in the template. Remove one of the spaces in both templates.